.gitignore

```diff
@@ -5,3 +5,5 @@ __pycache__/
 /builds/
 /misc/.tox/
 /misc/autodoc/downward-xmlrpc.secret
+*~
+#*
```

Comment:
```suggestion
```

Reply:
`#*` is also a pattern for emacs backup file.
For example, if I am editing `a.c`, it creates `#a.c` while I am still editing the file, and `*~` when the file is saved.

Reply:
I think both of these patterns should rather be in a user's global git configuration, not each project-specific repository. That way, users can configure these once for all projects, and the projects don't need to deal with all possible editor setups.

For what it's worth, as a fellow emacs user: I've configured emacs to put the *~ backups in a global place rather than alongside the original files because I think it's much less messy. And for the #* files, I think it's a feature *not* to have git ignore them because their presence indicates that there are unsaved edited files currently open, and having this shown clearly when doing a git status or similar prior to a commit is good. It's rarely a good idea to commit when there are unsaved changes.

Reply:
I agree that it would be better to leave this up to the user configuration then. But isn't `#` a comment in gitignore? Does this work as a pattern without escaping `#`?

Reply:
> put the *~ backups in a global place

This indeed works. I am fine with dropping this commit, it is just my habit from experience that plenty of students don't know these and commit absolute everything, and I got annoyed often enough removing .DS_store etc commited in a shared repo...

Reply:
I removed the gitignore commit.